ivy/functional/frontends/tensorflow/nn.py

```diff
@@ -150,8 +150,11 @@ def batch_normalization(x, mean, variance, offset, scale, variance_epsilon, name
 
 
 @to_ivy_arrays_and_back
-def dropout(x, prob, scale, dtype, name=None):
-    return ivy.dropout(x, prob, scale, dtype)
+def dropout(x, prob, scale, dtype, noise_shape):
+
+    ones = ivy.ones(noise_shape)
```

Comment:
You can just pass in `x` and `noise_shape` directly to `ivy.dropout` without doing this because it works in a similar way to tensorflow